frontend/src/tree/TreeItemContextMenu.tsx

```diff
@@ -44,60 +68,114 @@ export const TreeItemContextMenu = ({
     }
   };
 
+  const [deleteTreeItem, { loading: deleteTreeItemLoading, data: deleteTreeItemData, error: deleteTreeItemError }] =
+    useMutation<GQLDeleteTreeItemData, GQLDeleteTreeItemVariables>(deleteTreeItemMutation);
+
+  const deleteItem = () => {
+    const input: GQLDeleteTreeItemInput = {
+      id: uuid(),
+      editingContextId,
+      representationId: treeId,
+      treeItemId: item.id,
+    };
+    deleteTreeItem({ variables: { input } });
+    onClose;
```

Comment:
`onClose` is a function, use it as such `onClose()` otherwise it does not work. I'm pretty sure that if you test that, the context menu is never closed while deleting something